src/transformers/utils/hub.py

```diff
@@ -465,6 +465,11 @@ def cached_files(
                 "for this model name. Check the model page at "
                 f"'https://huggingface.co/{path_or_repo_id}' for available revisions."
             ) from e
+        elif isinstance(e, PermissionError):
+            raise OSError(
+                f"PermissionError at {e.filename} when  downloading {path_or_repo_id}. "
+                "Please check ownership and permissions of the cache directory."
```

Comment:
Given that there are common situations where this might happen, I would add them here to help users understand why this exception is appearing.

E.g. "(...) the cache directory. There are a few common explanations for this exceptions: 1) another user is downloading the same model (please wait); 2) another user as canceled the download of the model, and you can't acquire the lock (it may need to be manually removed)."